Template-2.md

```diff
@@ -0,0 +1,249 @@
+---
+layout: default
+title: Guidance Document Template
+permalink: /template/
+---
+# Contributing to the GSA FICAM Guides
```

Comment:
Remove this header
the default template use the Title as the first header in the page